crates/types/src/services/p2p/peer_reputation.rs

```diff
@@ -0,0 +1,56 @@
+/// PeerScore type used for Peer Reputation
+pub type PeerScore = f64;
+
+/// Minimum allowed peer score before peer is banned
+pub const MIN_PEER_SCORE: PeerScore = 0.0;
+/// Default value for peer score
+pub const DEFAULT_PEER_SCORE: PeerScore = 50.0;
```

Comment:
I think it would be better to use negative values for bad reputation and `0` as a default score for all.

Reply:
Sure, I will update it, but either way we will have to do another deep dive on scoring soon.